### PR TITLE
Payment Flow: visual flourishes

### PR DESCRIPTION
**Pull request by regiskuckaertz**
## Why are you doing this?

Introducing the final visual flourishes necessary for the pages to be complete, plus a few changes recommended by @Ap0c in a previous PR. In order to introduce the background image, I had to rework the markup a little by introducing a new parent `DIV` for the content (`.gu-content__content`).

Still need to polish a few things before this is 100% ready:

- [x] style background for browsers with no CSS grid support
- [x] style background on mobile
- ~~add images inside background bubbles (@Ap0c the `foreignObject` trick does not work, images in SVGs must be inserted with `image`)~~
- [x] tweak the subscribe button

[**Trello Card**](https://trello.com/c/YnKd7Kp5/717-npf-add-background#)

## Screenshots

screen shot 2018-09-03 at 09 43 05
screen shot 2018-09-03 at 09 43 15
screen shot 2018-09-03 at 11 03 42

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
Will leave the images in bubbles problem to the side for now, I want to get the redux part done ASAP as it is more important.

File: assets/pages/new-contributions-landing/components/ContributionSubmit.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ type PropTypes = {
 // ----- Render ----- //
 
 function ContributionSubmit(props: PropTypes) {
-  const currency: IsoCurrency = detect(props.countryGroupId);
+  const currency: IsoCurrency = fromCountryGroupId(props.countryGroupId) || 'GBP';
```
Comment (Mullefa) [inline assets/pages/new-contributions-landing/components/ContributionSubmit.jsx]: Out of interest, why are we no longer first looking for the currency in the query parameters?

**Comment by regiskuckaertz** [comment]
@Mullefa I added a screenshot of the mobile rendering. It is visible [here](https://support.theguardian.com/uk/contribute.new) and [here](https://support.theguardian.com/uk/thankyou.new) yes, but eventually will be visible only to those people in the variant bucket.

File: assets/pages/new-contributions-landing/components/ContributionSubmit.jsx (hunk 1 above)
Comment (regiskuckaertz) [reply assets/pages/new-contributions-landing/components/ContributionSubmit.jsx]: I wasn't aware, perhaps sth to pick-up when I'm done with the logic.

**Pull request merged**